pkgs/development/libraries/ucx/default.nix

```
@@ -1,7 +1,9 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen
-, numactl, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen, numactl
```

Please remove the `fetchpatch`. It is not used anymore.
```suggestion
{ lib, stdenv, fetchFromGitHub, autoreconfHook, doxygen, numactl
```